src/Phinx/Db/Table.php

```diff
@@ -703,11 +703,9 @@ protected function executeActions($exists)
         // Renaming a table is tricky, specially when running a reversible migration
         // down. We will just assume the table already exists if the user commands a
         // table rename.
-        $renamed = collection($this->actions->getActions())
-            ->filter(function ($action) {
-                return $action instanceof RenameTable;
-            })
-            ->first();
+        $renamed = current(array_filter($this->actions->getActions(), function ($action) {
+            return $action instanceof RenameTable;
+        }));
```

Comment:
foreach ($this->actions->getActions() as $action) {
    if ($action instanceof RenameTable) {
        $exists = true;
        break;
    }
}

Reply:
Yup, that's a way more efficient way of doing this. Missing the forest for the trees. 🤦 

I've also wrapped this with a `if (!$exists)` check as well.